mms/tests/integration_tests/test_utils.py

```diff
@@ -0,0 +1,193 @@
+    'shufflenet': 'https://s3.amazonaws.com/download.onnx/models/shufflenet.tar.gz',
+    'squeezenet': 'https://s3.amazonaws.com/download.onnx/models/squeezenet.tar.gz',
+    'vgg16': 'https://s3.amazonaws.com/download.onnx/models/vgg16.tar.gz',
+    'vgg19': 'https://s3.amazonaws.com/download.onnx/models/vgg19.tar.gz'}
+
+# models from onnx-mxnet model zoo
+onnx_mxnet_model_urls = {
+    'onnx-alexnet': 'https://s3.amazonaws.com/model-server/models/onnx-alexnet/alexnet.onnx',
+    'onnx-squeezenet': 'https://s3.amazonaws.com/model-server/models/onnx-squeezenet/squeezenet.onnx',
+    'onnx-inception_v1': 'https://s3.amazonaws.com/model-server/models/onnx-inception_v1/inception_v1.onnx',
+    'onnx-vgg19': 'https://s3.amazonaws.com/model-server/models/onnx-vgg19/vgg19.onnx'}
+
+# models from onnx-mxnet model zoo
+mxnet_model_urls = {
+    # TODO: check why these two models are failing github issue #345
+    'caffenet': 'https://s3.amazonaws.com/model-server/models/caffenet/caffenet.model',
+    'Inception-BN': 'https://s3.amazonaws.com/model-server/models/inception-bn/Inception-BN.model',
+    # 'lstm_ptb'        : 'https://s3.amazonaws.com/model-server/models/lstm_ptb/lstm_ptb.model',
+    'nin': 'https://s3.amazonaws.com/model-server/models/nin/nin.model',
+    'resnet-152': 'https://s3.amazonaws.com/model-server/models/resnet-152/resnet-152.model',
+    'resnet-18': 'https://s3.amazonaws.com/model-server/models/resnet-18/resnet-18.model',
+    # 'resnet50_ssd_model'    : 'https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model.model',
+    'resnext-101-64x4d': 'https://s3.amazonaws.com/model-server/models/resnext-101-64x4d/resnext-101-64x4d.model',
+    'squeezenet_v1.1': 'https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model',
+    'vgg19': 'https://s3.amazonaws.com/model-server/models/vgg19/vgg19.model',
+}
+
+
+def _download_file(download_dir, url):
+    """
+        Helper function to download the file from specified url
+    :param url: File to download
+    :return: None
+    """
+    try:
+        f = urlopen(url)
+        print("Downloading - {}".format(url))
+        with open(os.path.join(download_dir, os.path.basename(url)), "wb") as local_file:
+            local_file.write(f.read())
+    except HTTPError as e:
+        print("Failed to download {}. HTTP Error {}".format(url, e.code))
+    except URLError as e:
+        print("Failed to download {}. HTTP Error {}".format(url, e.reason))
+
+
+def cleanup(tmpdir):
+    print("Deleting all downloaded resources for SSD MXNet Model Server Integration Test")
+    shutil.rmtree(str(tmpdir))
+
+
+def create_model(
+        tmpdir,
+        url,
+        onnx_model,
+        onnx_source_model_zoo=True,
+        is_onnx_model=True):
+    # Download the files required for onnx integ tests.
+    download_dir = tmpdir + '/mxnet_model_server'
+    try:
+        if is_onnx_model:
+            os.mkdir(download_dir)
```

Comment:
for readability, move the body of the if is_onnx_model statement to a method -- e.g.
```
if is_onnx_model:
    create_onnx_model(...)
else:
   ...
```